easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-gsmpi-2017b.eb

```diff
@@ -0,0 +1,17 @@
+name = 'FFTW'
+version = '3.3.6'
+
+homepage = 'http://www.fftw.org'
+description = """FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
+ in one or more dimensions, of arbitrary input size, and of both real and complex data."""
+
+toolchain = {'name': 'gsmpi', 'version': '2017b'}
+toolchainopts = {'pic': True, 'optarch': False}
```

Comment:
@ocaisa Why disable `optarch`?

Reply:
Thats just a hangover from my earlier experiments to get the tests to pass. Removing and retesting now.